packages/ocular/src/services/organisation.ts

```diff
@@ -171,33 +170,35 @@ class OrganisationService extends TransactionBaseService {
                 installed_apps[webConnector_index].links = [];
               }
 
-              const linkExist = installed_apps[webConnector_index].links.find(
-                (ele) => ele.id === link_id
-              );
+              const linkExist = installed_apps[
```

Comment:
You are generating a new link id every time you call this function! link.id is not the best way to check if two links are similar.

Reply:
Skip the part of checking if inks are equal! Just add them to the array of links.